assert/assertions.go

```diff
@@ -721,7 +721,11 @@ func NotEqualValues(t TestingT, expected, actual interface{}, msgAndArgs ...inte
 func includeElement(list interface{}, element interface{}) (ok, found bool) {
```

Comment:
Something seems wrong with the docstring of this function... Do you mind updating the function name in line 717?

Reply:
yes, they are not synced. I fixed that and fixed further linting issues in the `assert` package.